### PR TITLE
Reduce mapping code

### PR DESCRIPTION
**Pull request by boidolr**
Instead of manually adding every new option one can pass the arguments through from the parsing code.
This simplifies adding additional command line options.

### CODE REVIEW

File: wordcloud/wordcloud_cli.py [hunk 1]
```diff
@@ -146,10 +140,16 @@ def parse_args(arguments):
...
         color_func = wc.get_single_color_func(args.color)
 
-    args.collocations = not args.no_collocations
-
     args.color_func = color_func
-    return args
+
+    args = vars(args)
+    # remove arguments not directly mapped
+    _ = args.pop('colormask', None)
```
Comment (amueller) [inline wordcloud/wordcloud_cli.py]: so these are just ignored? That seems odd.

**Comment by amueller** [comment]
This conflicts with #295, right? So merge this one first and adjust the other one?

**Comment by amueller** [comment]
otherwise looks good

File: wordcloud/wordcloud_cli.py (hunk 1 above)
Comment (boidolr) [reply wordcloud/wordcloud_cli.py]: No - they are used to choose the right color function some lines above and mapped to the same argument for the word cloud instance.

**Comment by boidolr** [comment]
It only conflicts insofar as there is one change in #295 which can be removed after this is merged ;)
I'll happily adapt the other PR.

File: wordcloud/wordcloud_cli.py (hunk 1 above)
Comment (amueller) [reply wordcloud/wordcloud_cli.py]: ah. maybe use ``args.pop('color')`` there so it's more obvious?

Comment (boidolr) [reply wordcloud/wordcloud_cli.py]: Right. Just using the `vars(args)` part earlier.

**Comment by boidolr** [comment]
Great I broke it. Will look at it again later and fix. Sorry for the inconvenience.

**Comment by amueller** [comment]
I merged the other one, but I think it'll be easy to fix here.

**Comment by boidolr** [comment]
Done & rebased. No other changes to the commits.

**Comment by amueller** [comment]
great, thanks :)

**Pull request merged**